src/marisa_trie.pyx

```diff
@@ -248,33 +254,37 @@ cdef class _Trie:
         self._trie.mmap(c_path)
         return self
 
-    def iterkeys(self, unicode prefix=""):
+    def iterkeys(self, prefix=NoPrefixGiven):
```

Comment:
This is one of the methods shared by unicode-based and bytes-based methods. If it's unicode-based, it needs to be a unicode string to be encoded as UTF8. If not, it needs to be a bytes object. So "" won't work nicely—would probably be misleading for people using BytesTrie.

`None` would work indeed. I just got used to a style of having explicitly-named defaults, this is a policy at the company I work for. If you want it, I'll change it after full review.